source/main.c

```diff
@@ -1,11 +1,170 @@
 // main.c 
 // folder home
 #include <stdio.h>
-int main(int argc, char const *argv[])
+#include <stdlib.h>
+ 
+typedef struct
 {
-    printf("hello world \n");
-    getchar();
-
+	// Initialize front and rear 
+	int rear, front;
+	// Circular Queue 
+	int size;
+	int* arr;
+}Queue;
+ 
+void create( int s, Queue* q)
+{
+	q->front = -1;
+	q->rear = -1;
+	q->size = s;
+	//maloc
+	int array[s];
+	if(((q->arr =(int*)malloc(sizeof(array)))) == NULL)
+	{
+		printf("Allocation error.");
+		exit(0);
+	}
```

Comment:
This is not the correct way to allocate memory.
Please, read this article about malloc https://www.programiz.com/c-programming/c-dynamic-memory-allocation.
You don't need to create an array[s], just enough determine which size you want allocate. In your case it would be (int*)malloc(s * sizeof(int))))

Reply:
done